Markdown version of the Academy Style Guide

DanZayas opened the pull request:
Adding my Homework to the read.me file.

DanZayas committed:
Adding my Homework to the read.me file

eocejo reviewed (commented):
Excellent work with this!
Just one correction. For the tables, you are setting one of the regular rows as the header. They would need to have headings instead.

DanZayas commented:
Hello!

I did notice it, but as we were asked to just copy the guide, I didn't want to add any new content.
That's why I was looking for an alternative to create a different chart setting, but I didn't find any.
I had tought about Do's and Dont's headings. Do you have any recommendations? 

Thanks for all the support and recommendations.

eocejo commented:
I believe it's a Markdown limitation that you can't have tables without
headers, although I don't remember if I've tried. If that's the case, then
you would be justified to add content since it is a restriction from the
tool.

Regarding the headings, for the first table, I would suggest something like
Abbreviation and Alternative since it is more descriptive.

Another option would be to see how a table with an empty header looks.
However, I think it is a good practice to always have headings in a table.


Eduardo Ocejo Vallin | WIZELINE

Technical Writer

***@***.*** | +52 33 16235624 (m)

Amado Nervo 2200, Esfera P6, Col. Jardines del Sol, 45050 Zapopan, Jal.


On Thu, Jun 3, 2021 at 10:53 AM DanZayas ***@***.***> wrote:

> Hello!
>
> I did notice it, but as we were asked to just copy the guide, I didn't
> want to add any new content.
> That's why I was looking for an alternative to create a different chart
> setting, but I didn't find any.
> I had tought about Do's and Dont's headings. Do you have any
> recommendations?
>
> Thanks for all the support and recommendations.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/DanZayas/tw-academy-s9/pull/1#issuecomment-853976912>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANPQ6QPALUIZJYFE5G6RWQ3TQ6QONANCNFSM455EWREA>
> .
>

-- 
*This email and its contents (including any attachments) are being sent to
you on the condition of confidentiality and may be protected by legal
privilege. Access to this email by anyone other than the intended recipient
is unauthorized. If you are not the intended recipient, please immediately
notify the sender by replying to this message and delete the material
immediately from your system. Any further use, dissemination, distribution
or reproduction of this email is strictly prohibited. Further, no
representation is made with respect to any content contained in this email.*

DanZayas committed:
Update README.md

DanZayas commented:
Great! 

Thanks for the tips.
I tried with the empty headers and it worked, but as you mentioned it is better to give those charts a title.
I'll keep all those recommendations in mind. 
Thanks for the guidance. 

I already added the titles and commited the changes.

DanZayas committed:
Update README.md

OMG!

I am so sorry, I don't know where my mind was.
4Cs! I apologize for not being concistent. 

Sorry for the slip and for making you waste your time. 

I already changed the headings from "Don't do this" and "Do this" to "Apostrophe" and "Alternative".
